src/lamplib/src/genny/tasks/auto_tasks.py

```diff
@@ -246,7 +246,7 @@ def all_tasks(self) -> List[GeneratedTask]:
         :return: all possible tasks irrespective of the current build-variant etc.
```

Comment:
Maybe we can add some documentation here like "except if the task has no AutoRun block"?